.changesets/feat_rreg_experimental_subgraph_metrics.md

```diff
@@ -0,0 +1,23 @@
+### [Subgraph Insights] Experimental Apollo Subgraph Fetch Histogram ([PR #8013](https://github.com/apollographql/router/pull/8013))
+
+This change adds a new, experimental histogram to capture subgraph fetch duration for Apollo Studio. The instrument,
+`apollo.router.operations.fetch.duration` has the following attributes:
+- client.name
+- client.version
+- has.errors
+- operation.id
+- graphql.operation.type
+- graphql.operation.name
+- subgraph.name
+
```

Comment:
This feels all like implementation details because the instrument is not actually exposed to the user at all.

Focusing on user impact, I think we should narrow this down to mention:
- We now optionally send subgraph request durations to GraphOS.
- Here's what that enables for users in Studio
- Here's how you enable the experimental configuration